l2beat/l2beat
# Remove discovery and link it to tools package

## What's changed
Migrated discovery to tools and linked it as a dependency

## How to test it?
Do the clean branch checkout and build.
Discovery should be working as if no changes were made.

## Sidenotes
It will be failing the CI until we publish tools release with migrated discovery 🥲 
Will tag anyone related to the topic as a reviewer.

[For tools changes see this tools pull request](https://github.com/l2beat/tools/pull/25)
